pkg/cmd/dockerregistry/dockerregistry.go

```diff
@@ -68,33 +71,33 @@ func Execute(configFile io.Reader) {
 	uuid.Loggerf = context.GetLogger(ctx).Warnf
 
 	// add parameters for the auth middleware
-	if config.Auth.Type() == server.OpenShiftAuth {
-		if config.Auth[server.OpenShiftAuth] == nil {
-			config.Auth[server.OpenShiftAuth] = make(configuration.Parameters)
+	if dockerConfig.Auth.Type() == server.OpenShiftAuth {
+		if dockerConfig.Auth[server.OpenShiftAuth] == nil {
+			dockerConfig.Auth[server.OpenShiftAuth] = make(configuration.Parameters)
 		}
-		config.Auth[server.OpenShiftAuth][server.AccessControllerOptionParams] = server.AccessControllerParams{
+		dockerConfig.Auth[server.OpenShiftAuth][server.AccessControllerOptionParams] = server.AccessControllerParams{
```

Comment:
Something wrong here, you removed field SafeClientConfig from AccessControllerParams.